### PR TITLE
Avatar alert

### PR DESCRIPTION
**Pull request by Sahil25061999**
Added avatar and alert components and basic styling.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,12 +4,27 @@
...
+          <li>option2</li>
+        </ul>
+      </header>
+      <nav class="sidebar">
+        <a href="component/avatar/avatar.html">Avatar</a>
+        <a href="component/alert/alert.html">Alert</a>
+      </nav>
+      <section class="component_info">
+        <h1>My component Library</h1>
+      </section> -->
```
Comment (rahulk31) [inline index.html]: Commented code can be removed.

File: component/alert/alert.css [hunk 2]
```diff
@@ -2,8 +2,9 @@
 
 .alert {
   width: max-content;
-  border-radius: 6px;
-  /* padding: 10px 100px 10px 15px; */
+  border-radius: 0.5rem;
+  width: 70%;
+  padding: 10px 100px 10px 15px;
   padding: 10px 50px;
```
Comment (rahulk31) [inline component/alert/alert.css]: Please use relative values.

File: index.html (hunk 1 above)
Comment (Sahil25061999) [reply index.html]: Thanks will do it

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+docu.css
+docu.js
```
Comment (rahulk31) [inline .gitignore]: file name can be improvised, upto you:)

**Review by rahulk31 — COMMENTED** [review]
Overall looks good to me. Can be merged.

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
Alerts generally have close button.
You can add 1 more alert with your secondary color of component Library.

**Review by purvasharmadev — APPROVED** [review]
Everything looks good, the code is clean, variable naming is nice, file names can be improved as suggested by Rahul, else good to go.

**Pull request merged**